cms/static/js/views/settings/main.js

```diff
@@ -131,12 +134,14 @@ var DetailsView = ValidatingView.extend({
         }, true));
     },
 
-    setupDatePicker: function (fieldName) {
+    setupDatePicker: function (fieldName, editable) {
         var cacheModel = this.model;
         var div = this.$el.find('#' + this.fieldToSelectorMap[fieldName]);
         var datefield = $(div).find("input.date");
         var timefield = $(div).find("input.time");
         var cachethis = this;
+        // ensure that false is saved if passed, all others default to true
+        var editable = editable !== false;
```

Comment:
What does the linter say about this (since the variable "editable" is already passed into the function)?